sky/backends/cloud_vm_ray_backend.py

```diff
@@ -325,8 +325,8 @@ def add_gang_scheduling_placement_group_and_setup(
             gpu_dict = {'GPU': acc_count}
             # gpu_dict should be empty when the accelerator is not GPU.
             # FIXME: This is a hack to make sure that we do not reserve
-            # GPU when requesting TPU.
-            if 'tpu' in acc_name.lower():
+            # GPU when requesting non-GPU accelerators (e.g., TPU/Inferentia).
+            if 'tpu' in acc_name.lower() or 'inferentia' in acc_name.lower():
```

Comment:
This seems not very generalizable for the cases where more "non-GPU" accelerators are supported, such as: `Trainium`. Is it possible that we can directly add a `--num-gpus` in the `ray start` command where we start the ray cluster at the following place to force the ray cluster to contain the missing `GPU`: https://github.com/skypilot-org/skypilot/blob/6fd5bf298f9d830bb3fbafee0e131ef688142f86/sky/provision/instance_setup.py#L211

Reply:
That seems reasonable. One concern is calling any accelerator "GPU" may be confusing to users (and it's only convenient for us).

Ray in recent releases took a different approach by not calling all accelerators "GPU". Instead there will be several resource names:
- https://github.com/ray-project/ray/blob/master/python/ray/_private/accelerators/neuron.py#L36 
- https://github.com/ray-project/ray/blob/master/python/ray/_private/accelerators/nvidia_gpu.py#L23
- https://github.com/ray-project/ray/blob/master/python/ray/_private/accelerators/tpu.py#L80
- etc.

Wdyt?

Reply:
I think this will only change the representation in `ray status`, which should ideally not be exposed to users, so not affecting the user side. That said, we can merge the current PR changes first to unblock our users first, and do a further refactoring in the future for avoiding the hardcode here (either by using `GPU` or further distinguish them).

For the hardcode, we probably want to add `trainium` as well. : )

Reply:
Sg, PTAL.